CHANGELOG.md

```
@@ -13,6 +13,12 @@ _June 03, 2026_
 - Open `ChatThreadListItemViewModel` for subclassing [#1482](https://github.com/GetStream/stream-chat-swiftui/pull/1482)
 - Allow injecting a custom channel list item view into `ChatChannelNavigatableListItem` to reuse the navigation logic [#1482](https://github.com/GetStream/stream-chat-swiftui/pull/1482)
 
+# [4.101.0](https://github.com/GetStream/stream-chat-swiftui/releases/tag/4.101.0)
+_June 03, 2026_
+
+### 🐞 Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)